apis/config/v1alpha2/configuration_types.go

```diff
@@ -39,8 +39,27 @@ type Configuration struct {
 	// unsuspended, they will start immediately.
 	ManageJobsWithoutQueueName bool `json:"manageJobsWithoutQueueName"`
 
-	// EnableInternalCertManagement controls whether to enable internal cert management or not.
+	// InternalCertManagement is configuration for internalCertManagement
+	InternalCertManagement InternalCertManagement `json:"internalCertManagement,omitempty"`
+}
+
+type InternalCertManagement struct {
+
+	// Enable controls whether to enable internal cert management or not.
 	// Defaults to true. If you want to use a third-party management, e.g. cert-manager,
 	// set it to false. See the user guide for more information.
-	EnableInternalCertManagement *bool `json:"enableInternalCertManagement,omitempty"`
+	Enable *bool `json:"enable,omitempty"`
+
+	// Namespace is the namespace where Service and Secret are deployed and
+	// is also used as part of DNSName.
+	// Defaults to kueue-system.
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
I added `Namespace` field to allow kueue-controller to be deployed to other namespaces (except kueue-system).
What do you think?

Reply:
It makes sense.

Reply:
Done.